core/lib/generators/solidus/install/templates/config/initializers/spree.rb.tt

```diff
@@ -40,6 +40,8 @@ Spree.config do |config|
   # Template to use when rendering layout
   # config.layout = "spree/layouts/spree_application"
 
+  # manually call associate_user instead of relying on current_order
+  # config.associate_user_in_authentication_extension = false
```

Comment:
As we're going to remove the setting in v4.0 altogether, I think it's better not to add it in the generated `spree.rb` for fresh installations. They should be using the new default, and there's no need for them to know anything about the old behavior that will be removed. Thoughts @kennyadsl ?

Reply:
I still would like some thought about it. @kennyadsl, what are we usually doing in these cases?

Reply:
I agree with @waiting-for-dev, I'd remove this as well.